test/addons-napi/test_general/test.js

```diff
@@ -34,3 +34,19 @@ assert.ok(test_general.testGetPrototype(baseObject) !==
 // test version management functions
 // expected version is currently 1
 assert.strictEqual(test_general.testGetVersion(), 1);
+
+[
+  123,
+  'test string',
+  function() {},
+  new Object(),
+  true,
+  undefined,
+  Symbol()
+].forEach((val) => {
+  assert.strictEqual(typeof val, test_general.testNapiTypeof(val));
```

Comment:
Oh. A tiny nit here and on line 52. I think the parameter order should be switched to reflect `actual, expected`